client/ayon_blender/api/pipeline.py

```diff
@@ -296,6 +296,15 @@ def set_resolution(data):
     scene.render.resolution_y = resolution_y
 
 
+def set_unit_scale(unit_scale_settings=None):
+    if unit_scale_settings is None:
+        return
+    unit_scale_enabled = unit_scale_settings.get("enabled")
+    if unit_scale_enabled:
+        unit_scale = unit_scale_settings.get("base_file_unit_scale")
+        bpy.context.scene.unit_settings.scale_length = unit_scale
```

Comment:
I think this being a function is redundant. It's odd with this argument - I'd expect it to be the scale you want to set, regardless of whether a setting is enabled or not.